Fix subscriptions when referer is not available

amiedes opened the pull request:
Fixes rollbar [#1202](https://rollbar.com/Populate/gobierto/items/1202/occurrences/49905012625/)

## What does this PR do

When creating a new subscription, if the referer is not available it redirects to the home page.

Commit:
Fix subscriptions when referer is not available

Fixes rollbars like https://rollbar.com/Populate/gobierto/items/1202/occurrences/49905012625/

Commit:
Merge branch 'master' into rollbar-1202

 Conflicts:
	test/integration/user/subscription_create_test.rb

ferblape commented on app/controllers/user/subscriptions_controller.rb:
```diff
@@ -50,8 +50,7 @@ def create
         end
-
-        redirect_to request.referrer
+        redirect_to(request.referrer || root_path)
```
There's a new way to do this!

https://blog.bigbinary.com/2016/02/29/rails-5-improves-redirect_to_back-with-redirect-back.html

ferblape commented on app/controllers/meta_welcome_controller.rb:
```diff
@@ -7,7 +7,7 @@ def index
 
     if current_site.configuration.home_page != "GobiertoCms"
-      redirect_to eval("#{current_site.configuration.home_page.underscore}_root_path")
+      redirect_to(current_site.root_path)
```
👍

Commit:
Use new syntax for redirecting to previous page

https://blog.bigbinary.com/2016/02/29/rails-5-improves-redirect_to_back-with-redirect-back.html

amiedes commented:
Done 💅
